applications/asset_tracker_v2/src/cloud/cloud_codec/json_common.c

```diff
@@ -560,6 +560,158 @@ int json_common_ui_data_add(cJSON *parent,
 	return err;
 }
 
+int json_common_neighbor_cells_data_add(cJSON *parent,
+					struct cloud_data_neighbor_cells *data,
+					enum json_common_op_code op)
+{
+	int err;
+
+	if (!data->queued) {
+		return -ENODATA;
+	}
+
+	err = date_time_uptime_to_unix_time_ms(&data->ts);
+	if (err) {
+		LOG_ERR("date_time_uptime_to_unix_time_ms, error: %d", err);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_MCC, data->cell_data.current_cell.mcc);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_MNC, data->cell_data.current_cell.mnc);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_CID, data->cell_data.current_cell.id);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_TAC, data->cell_data.current_cell.tac);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_EARFCN,
+			      data->cell_data.current_cell.earfcn);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_TIMING,
+			      data->cell_data.current_cell.timing_advance);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		return err;
+	}
+
+	err = json_add_number(parent, DATA_NEIGHBOR_CELLS_RSRP,
+			      data->cell_data.current_cell.rsrp);
```

Comment:
RSRP reported from the modem needs to be adjusted like this: https://github.com/nrfconnect/sdk-nrf/blob/aad5b21ff8261962580826321695330d4f8977f7/lib/multicell_location/services/skyhook_integration.c#L141-L148

Reply:
Nice. That works for me.

Reply:
Oh, I haven't noticed that we send raw RSRP values before. The underlying AT commands differ a bit in mapping of RSRP and RSRQ values. For cell measurements, RSRP and RSRQ can be negative: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/ncellmeas_set.html

The offsets in the Skyhook integration that I linked above seem to be off by 1. It should be 139 for negative inputs and 140 for positive ones. 
We should be consistent and either normalize all RSRP values on the device or in the cloud, and I'm fine with either.

Reply:
I think what can be done on the cloud side should be done on the cloud side, but if this normalization has no noticable impact on the device feel free to convert it there.

Reply:
This really doesn't impose any impact on the device. Converting these values on the device could create just a little bit less work for customers wanting to use their own cloud. (And doesn't want to bother depending on low level AT stuff). Ill add normalization of the values for both cases.